rpcs3/rpcs3qt/emu_settings.cpp

```diff
@@ -262,8 +262,14 @@ bool emu_settings::ValidateSettings(bool cleanup)
 	return is_clean;
 }
 
-void emu_settings::SaveSettings() const
+void emu_settings::SaveSettings()
 {
+	if (!m_title_id.empty())
```

Comment:
you can move the config_name code up and put your code into the else path

Reply:
Done.